api/v1/verticadb_types.go

```diff
@@ -354,6 +361,27 @@ const (
 	CommunalInitPolicyCreateSkipPackageInstall = "CreateSkipPackageInstall"
 )
 
+// RestorePointPolicy is used to locate the exact archive and restore point within archive
+// when a database restore is intended
+type RestorePointPolicy struct {
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors="urn:alm:descriptor:com.tectonic.ui:advanced"
```

Comment:
`RestorePoint` in VerticaDBSpec already has the xDescriptor `ui:advanced` marker so you don't need to add it again to each of its subfield. `// +operator-sdk:csv:customresourcedefinitions:type=spec` is enough.